ts/Data/DataTable.ts

```diff
@@ -1382,6 +1388,16 @@ class DataTable extends DataTableCore implements DataEvent.Emitter<DataTable.Eve
             rows
         });
     }
+
+    /**
+     * Returns the medified (clone) or the original data table.
+     *
+     * @return {Highcharts.DataTable}
+     * The medified (clone) or the original data table.
+     */
+    public getModified(): DataTable {
+        return this.modified || this;
+    }
```

Comment:
DataTable is a child of DataTableCore, so we don't need to have it both places if we have it in parent.